src/editors/vscode/scripts/bogo.py

```diff
@@ -6,7 +6,7 @@
 
 
 BORING_REPO = "https://github.com/randombit/boringssl.git"
-BORING_BRANCH = "rene/runner-20240524"
+BORING_BRANCH = "rene/runner-20241016"
```

Comment:
This config is duplicated thrice in the repository: [I had said we should centralize this](https://github.com/randombit/botan/pull/4359#discussion_r1792331001).

Suggestion:

Lets add a file in `src/configs/repo_config.env` that can contain such configurations. Side note: Personally, I'd probably add it as a root-level file (eg. `.repo_config.env`), but I remember that you wanted to keep the repo root as tidy as possible.

This file could then be consumed by `.github/workflows/ci.yml` and `.../nightly.yml` as well as here and (for now) just contain:

```txt
# the fork of boringssl that should be used for BoGo tests
BORINGSSL_REPO=https://github.com/randombit/boringssl.git

# the branch in our fork of boringssl that should be used for BoGo tests
BORINGSSL_BRANCH=rene/runner-20241016
```

We might have more such configuration that is worth to centralize like that. In any case: I'd do this adaption in an independent PR (likely after 3.6.0).

What are your thoughts on this?

Reply:
> Lets add a file in src/configs/repo_config.env that can contain such configurations. 

Sure sounds good.

> Side note: Personally, I'd probably add it as a root-level file (eg. .repo_config.env), but I remember that you wanted to keep the repo root as tidy as possible.

Indeed so. Thank you for accomodating my autistic needs on this topic.